strato

```diff
@@ -66,6 +66,7 @@ VAULT_URL                   - (default: BlockApps Vault) The URL of STRATO Vault
 
 Marketplace App variables:
 MP_IS_BOOTNODE                    - (default: false) [true|false] Whether or not to run the Markeplace in boot node mode or connect to the existing marketplace deployment based on address provided in MP_DAPP_ADDRESS
+DISABLE_MARKETPLACE               - (default: false) [true|false] Whether or not to run the Marketplace backend & UI containers
```

Comment:
I am okay with this name, but may be MP_DISABLED would be more consistent with the other vars